src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -546,6 +549,183 @@ public ActionResult OtherTasksPanel(AccountDashboardViewModel model)
             return PartialView(model);
         }
 
+        [HttpGet]
+        [Route("whichcourseyourapprenticewilltake")]
+        public ActionResult WhichCourseYourApprenticeWillTake()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        [Route("whichcourseyourapprenticewilltake")]
+        [ValidateAntiForgeryToken]
+        public ActionResult WhichCourseYourApprenticeWillTake(string choice)
+        {
+            switch (choice)
+            {
+                case Yes:
+                {
+                    return RedirectToAction(ControllerConstants.HaveYouChosenATrainingProviderActionName);
+                }
+
+                case No:
+                {
+                    return RedirectToAction(ControllerConstants.YouCannotSetupAnApprenticeshipYetCourseProviderActionName);
+                }
+
+                default:
+                {
+                    var model = new
+                    {
+                        InError = true
+                    };
+
+                    return View(model);
+                }
+            }
+        }
+
+        [HttpGet]
+        [Route("youcannotsetupanapprenticeshipyetcourseprovider")]
+        public ActionResult YouCannotSetupAnApprenticeshipYetCourseProvider()
+        {
+            return View();
+        }
+
+        [HttpGet]
+        [Route("haveyouchosenatrainingprovider")]
+        public ActionResult HaveYouChosenATrainingProvider()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        [Route("haveyouchosenatrainingprovider")]
+        [ValidateAntiForgeryToken]
+        public ActionResult HaveYouChosenATrainingProvider(string choice)
+        {
+            switch (choice)
+            {
+                case Yes:
+                {
+                    return RedirectToAction(ControllerConstants.WillApprenticeshipTrainingStartActionName);
+                }
+
+                case No:
+                {
+                    return RedirectToAction(ControllerConstants.YouCannotSetupAnApprenticeshipYetProviderActionName);
+                }
+
+                default:
+                {
+                    var model = new
+                    {
```

Comment:
Can an error be added to the ModelState and the existing error controls used?

Reply:
Done